go/teams/member_set.go

```diff
@@ -31,42 +31,42 @@ func newMemberSet(ctx context.Context, g *libkb.GlobalContext, req keybase1.Team
 
 func (m *memberSet) loadMembers(ctx context.Context, g *libkb.GlobalContext, req keybase1.TeamChangeReq) error {
 	var err error
-	m.Owners, err = m.loadGroup(ctx, g, req.Owners)
+	m.Owners, err = m.loadGroup(ctx, g, req.Owners, true)
```

Comment:
I know it's not in this PR, but should `member` live in AVDL so @mmaxim can access it?